function/pkg/resources/unstructured/function.go

```diff
@@ -0,0 +1,100 @@
+package unstructured
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path"
+
+	"github.com/kyma-incubator/hydroform/function/pkg/workspace"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+type ReadFile = func(filename string) ([]byte, error)
+
+const functionApiVersion = "serverless.kyma-project.io/v1alpha1"
+
+var errUnsupportedSource = fmt.Errorf("unsupported source")
+
+func NewFunction(cfg workspace.Cfg) (unstructured.Unstructured, error) {
```

Comment:
Same as with the other unstructured, why not use https://github.com/kyma-project/kyma/blob/8e0d3c51dd7af6bbd8aaafe825baf9aee01701b9/components/function-controller/pkg/apis/serverless/v1alpha1/function_types.go#L25 ?

Reply:
https://github.com/kyma-incubator/hydroform/pull/129#discussion_r498808362